concepticondata/conceptlists.tsv

```diff
@@ -217,5 +217,5 @@ Auracher-2017-16	Auracher, Jan	2017		16	acquisition	English	Global	https://www.n
 Pagel-2018-200	Pagel, Mark AND Meade, Andrew	2018		200	basic	English	Global	http://rstb.royalsocietypublishing.org/highwire/filestream/38234/field_highwire_adjunct_files/0/rstb20160517supp1.pdf	Pagel2018		The list is based on the [210-item list of ABVD](:ref:Blust-2008-210), the  [207-item list of IELex](:ref:Dunn-2012-207), and the [100-item list by Grollemund et al.](:ref:Grollemund-2015-100). The RATE column describes the lexical replacement rate of an item per year, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund et al. (2015)](:bib:Grollemund2015), whereas, to the best of our knowledge, the list only contains 100 meanings.		
 Davies-1985-116	Davies, John AND Comrie, Bernard	1985		116	basic,areal	English	New Guinea	http://sealang.net/archives/pl/pdf/PL-A63.275.pdf	Davies1985		A 116 item wordlist survey of the Upper Yuat region, based on a variant of survey lists by the Summer Institute of Linguistics (currently not in Concepticon yet).		
 Bengtson-2017-12	Bengtson, John D	2017		12	basic	English	Global		Bengtson2017		Twelve basic word meanings from the [Swadesh list of 100 items](:ref:Swadesh-1955-100).		
-Kibrik-2012-122	Kibrik, Andrej D.	2012		122	basic	English	Global		Kibrik2012		A list of 122 verbs (based on Tomasello 1992) used for comparing verbal lexical meaning construction across (primarily) Koyukon, Russian, and English.		
+Kibrik-2012-122	Kibrik, Andrej D.	2012		122	basic	English	Global		Kibrik2012		A list of 122 verbs used for comparing verbal lexical meaning construction across (primarily) Koyukon, Russian, and English.		
```

Comment:
Given that we'll definitely NEED Tomasello to further dig intto the origin of htis list, I'd actually prefer to have it included already as a reference, so that it is easier to add it later to concepticon (do we already have an issue for Tomasello?)

Reply:
Ok, will add. I've got the list here with me (the appendix of the book is something like 90 pages though, would have to check where exactly the  verbs come from).

Reply:
once this is done, feel free to merge, I'll add the Russian concepts later then. We'll then also need a strategy to add Tomasello...